### ISSUE TITLE
Medical Emergency Service Hotline 116 117 not linked as phone in Request TAN

### ISSUE THREAD

**Issue by MikeMcC399** [corona-warn-app/cwa-app-android#2291]
## Avoid duplicates
* [X] Bug is not mentioned in the [FAQ](https://www.coronawarn.app/en/faq/)
* [X] Bug is specific for Android only, for general issues / questions that apply to iOS and Android please raise them in the [documentation repository](https://github.com/corona-warn-app/cwa-documentation)
* [X] Bug is not already reported in another issue

## Technical details

- Device name: Samsung Galaxy A5 (2017) SM-A520F
- Android version: 8.0.0
- App version: 1.11.0 (also checked with branch release/1.12.x)

## Describe the bug

In the Request TAN view the telephone number 116 117 is not linked as a phone number.
For comparison, the same phone number is linked in the view shown via App Information > Technical hotline.

## Steps to reproduce the issue

Note: autoLink used in CWA version 1.11.0 is dependent on the device's locale setting. The following steps are with the device's language set to English (United Kingdom). If the device's language is set to English (United States) the telephone number 116 117 is not linked in the Technical Hotline view nor in the Request TAN view.

1. Under "Have you been tested?" tap "NEXT STEPS"
2. Scroll down to "No TAN yet?" and tap
3. Scroll down to the paragraph "If you have any health-related questions, please contact your general practitioner or the medical emergency service hotline, telephone: 116 117."
4. Tap on the phone number 116 117 and note that nothing happens.

![Request TAN](https://user-images.githubusercontent.com/66998419/107015328-e9342580-679c-11eb-9920-7dfbfc8936d1.jpg)

---
Compare to the link to phone number 116 117 shown on the Technical hotline view

![Technical hotline](https://user-images.githubusercontent.com/66998419/107015491-12ed4c80-679d-11eb-8e45-e93939c0738d.jpg)

## Expected behaviour

After opening the view Request TAN through "Have you been tested?" > "NEXT STEPS" > "No TAN yet?" the phone number 116 117 should be underlined and tapping on it should bring up the Phone app.

The Request TAN view should show the phone number 116 117 the same way that the Technical Hotline view (accessed through three-dot overview menu > App Information > Technical hotline) shows it.

## Possible Fix

Add the following attributes to 
TextView +id/submission_contact_body_other
in layout\include_submission_contact.xml

so that the TextView is aligned with 
TextView @+id/information_contact_body_other
in layout\fragment_information_contact.xml
```
android:autoLink="phone"
android:textColorLink="@color/colorTextTint"

```

~~I will submit a PR for this.~~
PR https://github.com/corona-warn-app/cwa-app-android/pull/2292

## Additional context

This came to light due to examining phone number autoLinking under https://github.com/corona-warn-app/cwa-app-android/issues/2275

---
Internal tracking ID: [EXPOSUREAPP-5007](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-5007)
This issue will be solved with EXPOSUREAPP-4992.

**Comment by dsarkar** [comment]
Internal tracking ID: [EXPOSUREAPP-5007](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-5007)

**Comment by MikeMcC399** [comment]
This issue is mitigated by PR #2286, which is pending merge into the release/1.12.x branch.

#2292 proposed to add the autoLink attribute for the Request TAN view and this only solved the issue for locale English (United Kingdom). If the locale was set to English (United States) then autoLink did not work for either the Technical Hotline nor for the Request TAN view.

#2286 works for locale English (United States) as well, so it is to be preferred over #2292.

PR #2292 will be closed in favour of #2286.

**Comment by dsarkar** [comment]
@MikeMcC399 Thanks for your contributions in any case! We keep this issue open until we see the fix in the release. 

This issue will be solved with [EXPOSUREAPP-4992](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-4992).

**Comment by MikeMcC399** [comment]
PR #2286 (EXPOSUREAPP-4992) is now in the [1.13.0 Milestone](https://github.com/corona-warn-app/cwa-app-android/milestone/19).

**Comment by MikeMcC399** [comment]
Confirmed fixed in CWA 1.13.2 release.

## Steps to verify fix
1. Set device language to English (United States) 
2. Under "Have you been tested?" tap "NEXT STEPS"
3. Scroll down to "No TAN yet?" and tap
4. Scroll down to the paragraph "If you have any health-related questions, please contact your general practitioner or the medical emergency service hotline, telephone: 116 117."
5. Tap on the phone number 116 117 and note that the phone app is called up.
6. Return to top level of app
7. Tap three-dot icon > App Information > Technical Hotline
8. Scroll down to bottom where phone number 116 117 is listed
9. Tap on the phone number 116 117 and note that the phone app is called up.

**Comment by dsarkar** [comment]
@MikeMcC399 Thank you.
